surya/detection/loader.py

```diff
@@ -15,9 +15,16 @@ def __init__(self, checkpoint: Optional[str] = None):
         super().__init__(checkpoint)
 
         if self.checkpoint is None:
-            self.checkpoint = settings.DETECTOR_MODEL_CHECKPOINT
+            self.checkpoint = [settings.DETECTOR_MODEL_CHECKPOINT, settings.INLINE_MATH_MODEL_CHECKPOINT]
+        
+        checkpoints, revisions = [], []
```

Comment:
This would be a lot better as a separate loader that inherits from the detectionloader versus loading everything twice.  The one predictor == one model paradigm is very nice